lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1646,6 +1658,18 @@ def _get_expected_cmd(self):
             return None
         return shlex.split(self.parameters.command)
 
+    def _get_expected_tmpfs(self):
+        self.log('_get_expected_tmpfs')
+        result = dict()
```

Comment:
missing tmpfs needs to be handled, this module fails with a "NoneType is not iterable" error when a tmpfs attribute is not provided